ci_jobs_trigger/libs/openshift_ci/ztream_trigger/zstream_trigger.py

```diff
@@ -160,7 +162,7 @@ def process_and_trigger_jobs(logger: logging.Logger, version: str | None = None)
             logger.info(
                 f"{LOG_PREFIX} New Z-stream version {_latest_version}:{_version_channel} found, triggering jobs: {_jobs}"
             )
-            if trigger_jobs(config=config, jobs=_jobs, logger=logger):
+            if trigger_jobs(config=config, jobs=_jobs, zstream_version=_latest_version, logger=logger):
```

Comment:
[ditto](https://github.com/RedHatQE/ci-jobs-trigger/pull/165#discussion_r1615894156)

Reply:
sure @omrirh I'll update this, but since we are using keyword argument here, I believe it's not necessary.

Reply:
@omrirh fixed the typo and updated the code